src/table/types.ts

```diff
@@ -94,6 +94,7 @@ export interface HandleBodyKeyDownProps {
   totalsPosition: TotalsPosition;
   paginationNeeded: boolean;
   selectionsAPI: ExtendedSelectionAPI;
+  isFlagEnabled?(flag: string): boolean;
```

Comment:
This one shouldn't be optional. We cast to this and then we know that these props exist